### DISCUSSION TITLE
Welcome to starquant Discussions!

### DISCUSSION THREAD

**Discussion by physercoe** [physercoe/starquant#9] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by useric** [comment]
is there any wechat or QQ group?

**Reply by physercoe** [reply]
not yet~

**Comment by foo126** [comment]
能不能放一个安装版？

**Comment by physercoe** [comment]
这段时间看看准备一个docker镜像出来，到时可以通过docker运行，如果不想自己手动安装的话。
